frameworks/kafka/docs/configure.md

```diff
@@ -194,6 +195,28 @@ The name of this Kafka instance in DC/OS. This is an option that cannot be chang
 *   **In DC/OS CLI options.json**: `name`: string (default: `kafka`)
 *   **DC/OS web interface**: The service name cannot be changed after the cluster has started.
 
+## Kill Grace Period
+
+The number of seconds to grant each broker to cleanly shutdown in response to SIGTERM.
```

Comment:
"The kill grace period is the number of seconds each broker has to cleanly shut down in response to SIGTERM. If a broker exceeds this time, it will be killed."

(the idea here is to provide a bit more high-level description of the option)